app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,44 @@
+class CommentsController < ApplicationController
+  # def create
+  #   @comment = current_user.comments.new(comment_params)
+
+  #   respond_to do |format|
+  #     if @comment.save
+  #       format.html { redirect_to user_posts_path(current_user) }
+  #     else
+  #       format.html { render :show }
+  #     end
+  #   end
+  # end
+
+  # private
+
+  # def comment_params
+  #   params.require(:comment).permit(:text)
+  # end
```

Comment:
- [ ] Let's remove commented code to make the project look clean and readable

Reply:
Thanks am gonna work on it asap